src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -9008,6 +9011,56 @@ public static String appendIfMissingIgnoreCase(final String str, final CharSeque
         return appendIfMissing(str, suffix, true, suffixes);
     }
 
+    /**
+     * <p>Returns either the passed in String with the specified suffix attached,
+     * or if the String is empty ("")/{@code null}, an empty string.</p>
+     *
+     * <p>Whitespace is defined by {@link Character#isWhitespace(char)}.</p>
+     *
+     * <pre>
+     * StringUtils.appendIfNotBlank(null, " ")      = ""
```

Comment:
The javadoc needs an update to be in sync with the method name.

Reply:
Fixed in latest commit.